providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/grpc.py

```diff
@@ -57,24 +58,40 @@ def __init__(
         self.deadline = config.deadline * 0.001
         self.connected = False
 
+        self._cache: typing.Optional[BaseCacheImpl] = (
+            LRUCache(maxsize=self.config.max_cache_size)
+            if self.config.cache_type == CacheType.LRU
+            else None
+        )
+
     def _create_stub(
         self,
     ) -> typing.Tuple[evaluation_pb2_grpc.ServiceStub, grpc.Channel]:
         config = self.config
         channel_factory = grpc.secure_channel if config.tls else grpc.insecure_channel
         channel = channel_factory(
             f"{config.host}:{config.port}",
-            options=(("grpc.keepalive_time_ms", config.keep_alive_time),),
+            options=(("grpc.keepalive_time_ms", config.keep_alive),),
         )
         stub = evaluation_pb2_grpc.ServiceStub(channel)
         return stub, channel
 
     def initialize(self, evaluation_context: EvaluationContext) -> None:
         self.connect()
+        self.retry_backoff_seconds = 0.1
+        self.connected = False
```

Comment:
this should be moved before we try to connect (line 80)

Reply:
thank you, this was a merge error, it is resolved now